src/initramfs/cmd/init/pkg/mount/mount.go

```diff
@@ -48,27 +48,34 @@ type Point struct {
 
 // BlockDevice represents the metadata on a block device probed by libblkid.
 type BlockDevice struct {
-	dev   string
-	TYPE  string
-	UUID  string
-	LABEL string
+	dev       string
+	TYPE      string
+	UUID      string
+	LABEL     string
+	PARTLABEL string
+	PARTUUID  string
 }
 
-// Init initializes the mount points.
-func Init(s string) (err error) {
-	once.Do(func() {
```

Comment:
I can't remember why I made this a singleton. We may have to revert to using `once.Do`. But I'm not sure at this point. We can revisit this after this PR is no longer a WIP.